kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -85,3 +85,33 @@ $ scaleCluster.sh -d domain1 -n weblogic-domain-1 -c cluster-1 -r 3
 domain.weblogic.oracle/domain1 patched
 [2021-02-26T19:04:14.466 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
 ```
+
+### Script to view the status of any WebLogic cluster
+
+The `clusterStatus.sh` script can be used to view the status of any WebLogic cluster in the WebLogic domain managed by WKO. The status contains information about the minimum, maximum, goal, current, and ready replica count for a WebLogic cluster. This script dumps a table containing the status for WebLogic cluster(s) in one or more domains across one or more namespaces. See the script `usage` information by using the `-h` option.
```

Comment:
I don't think we've generally used the "WKO" acronym in our documentation and instead just said "the operator" or "WebLogic", as appropriate. In this line, I think it's "the operator" and then later, "WebLogic Cluster Status."

Reply:
Definitely true. We do not use the acronym WKO in the docs; let's not start (there are enough acronyms in this world). Much better to use the term, "the operator," as Ryan suggested.